dgraph/cmd/alpha/run.go

```diff
@@ -263,8 +263,8 @@ they form a Raft group and provide synchronous replication.
 
 	flag.String("feature-flags", worker.FeatureFlagsDefaults, z.NewSuperFlagHelp(worker.FeatureFlagsDefaults).
 		Head("Feature flags to enable various experimental features").
-		Flag("list-in-normalize", "enables returning a list when there are multiple fields with same alias, "+
-			"see here for more details https://github.com/dgraph-io/dgraph/pull/7639").
+		Flag("normalize-response", "chose the kind of response for @normalize when there are multiple fields with same "+
```

Comment:
Doesn't Flag have an option to indicate what is the default value?

Reply:
Done

Reply:
```
      --feature-flags list         Feature flags to enable various experimental features
                                       normalize-response=list; configure @normalize formatting.  list: (default) fields with the same alias are returned as a list; 'flat': returns a single value for fields with same alias.For more details, see https://github.com/dgraph-io/dgraph/pull/7639
                                    (default "normalize-response=list")
```